service/src/main/java/bio/terra/pipelines/app/controller/PipelineRunsApiController.java

```diff
@@ -198,7 +198,12 @@ public ResponseEntity<ApiGetPipelineRunsResponse> getAllPipelineRuns(
                     new ApiPipelineRun()
                         .jobId(pipelineRun.getJobId())
                         .status(pipelineRun.getStatus().name())
-                        .description(pipelineRun.getDescription()))
+                        .description(pipelineRun.getDescription())
+                        .timeSubmitted(pipelineRun.getCreated().toString())
+                        .timeCompleted(
+                            pipelineRun.getStatus().isCompleted()
+                                ? pipelineRun.getUpdated().toString()
+                                : null))
```

Comment:
does setting it to null change the default behavior of not being in the response if there is no value?  I'm pretty sure it doesnt since its starts out as `null` but just curious

Reply:
this has the same effect as just not setting timeCompleted at all if the run isn't completed. here's what's in the API response: 
```
    {
      "jobId": "30d0ec3d-06e0-41ef-944a-846704926037",
      "status": "RUNNING",
      "description": "local test jobmapkeys refactor and hooks, fixed wdl version",
      "timeSubmitted": "2024-10-08T13:34:39.289930"
    },
```
do you prefer that we not set it, rather than set it to null? it seems like it might make the logic a little gnarlier since I don't know if you can call a `.timeCompleted()` inside an `if` statement?

Reply:
nope this is great, i was just double checking the response was the same even though it should have been obvious to me that it was